README.adoc

```diff
@@ -197,6 +197,18 @@ $ mvn clean install -Pdistribution,sample
 Note that none of those currently ships with the UI. Follow the instructions on
 link:https://github.com/hawkular/hawkular-ui-components[hawkular-ui-components] in order to run the UI.
 
+== Releasing
+
+To perform a release of Hawkular Accounts, you'll need push
+permissions to the GitHub repository and execute the following:
+
+[source,bash]
+----
+mvn release:prepare release:perform -Psample,distribution
```

Comment:
While it is good to have those instructions, could we come to a mode where for all Hawkular modules the command is the very same to release?